client/src/components/Blog.js

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect, useState } from "react";
+
+function Blog() {
+  const [blog, setBlog] = useState([]);
+
+  useEffect(() => {
+    const getBlogs = async () => {
+      try {
+        const res = await fetch("https://balancebox.onrender.com/get-blog");
```

Comment:
Put `https://balancebox.onrender.com` in `.env`, it's better to keep your domain link stored in a variable and use it, later when there is a change in domain it becomes easier to change.